R/pkg/R/DataFrame.R

```diff
@@ -170,7 +170,7 @@ setMethod("isLocal",
 setMethod("showDF",
           signature(x = "DataFrame"),
           function(x, numRows = 20) {
```

Comment:
Could you also make this a `pretty = TRUE` here and pass in `pretty` as the argument in `callJMethod` ? The code would look something like 
```
function(x, numRows = 20, pretty = TRUE) {
  s <- callJMethod(x@sdf, "showString", numToInt(numRows), pretty)
}
```